core/src/Torch/Raw/Storage.hs

```diff
@@ -51,3 +60,23 @@ class THStorage t where
   p_free :: FunPtr (Ptr t -> IO ())
   p_resize :: FunPtr (Ptr t -> CPtrdiff -> IO ())
   p_fill :: FunPtr (Ptr t -> CDouble -> IO ())
+
+
+-- TODO: Complete the implementations
```

Comment:
Can you either complete these in this PR or file a ticket and remove this TODO? TODOs tend to get lost and stale (even though I am probably a purveyor as well).